internal/args/errors.go

```diff
@@ -187,3 +187,19 @@ func missingIndices(index, length int) string {
 	}
 	return strings.Join(s, ",")
 }
+
+type CannotParseDateError struct {
+	ArgValue               string
+	AbsoluteTimeParseError error
+	RelativeTimeParseError error
+}
+
+func (e *CannotParseDateError) Error() string {
```

Comment:
I think I would prefer to return a much simple error and handle the pretty error in the core package